source/codegen/templates/service_helpers.mako

```diff
@@ -19,7 +21,11 @@ ${initialize_input_params(function_name, parameters)}
       };
       uint32_t session_id = 0;
       const std::string& session_name = request->session_name();
+      %if service_helpers.has_close_function(function,function_name):
+      auto cleanup_lambda = ${function[function_name].get('close_function', '')};
```

Comment:
I believe you can change the `get('close_function', '')` to `['close_function']. Before we got the if statement evaluating properly, we were using the get to deal with methods that didn't have a 'close_function'. But now that the if above it is evaluating we should be able to just use bracket notation to get the 'close_function'.